.env.template

```
@@ -66,6 +66,7 @@ S3_AWS_ACCESS_KEY_ID=
 S3_AWS_SECRET_ACCESS_KEY=
 ELEMENTARY_S3_BUCKET=
 LATEST_ELEMENTARY_PACKAGE_VERSION=
+S3_IMAGES_BUCKET=                     # shared bucket for all image uploads (logos, etc.)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)